packages/devtools-client-adapters/src/firefox/commands.js

```diff
@@ -71,38 +98,39 @@ function onNewBreakpoint(location, res) {
     column: actualLocation.column
   } : location;
 
-  return BreakpointResult({
+  return {
     id: bpClient.actor,
-    actualLocation: Location(actualLocation)
-  });
+    actualLocation
+  };
 }
 
-function removeBreakpoint(breakpointId) {
+function removeBreakpoint(breakpointId: BreakpointId) {
   const bpClient = bpClients[breakpointId];
-  bpClients[breakpointId] = null;
+  delete bpClients[breakpointId];
```

Comment:
flow did not like setting this to null so I changed to delete, we should really move to using a `Set` here though.